docs/administrative/change_account_name.md

```diff
@@ -0,0 +1,64 @@
+---
+permalink: change-account-company-name
+audit_date: '2018-03-13'
+title: Change an account's company name
+type: article
+created_date: '2018-03-13'
+created_by: Rackspace Support
+last_modified_date: '2022-07-04'
+last_modified_by: Rohit Rajput 
+---
+
+Change your account's company name
+====================================
+
+Complete a **Service Transfer** form if your company has filed for a name change with the Secretary of State where the company is registered, and the tax ID is changed.
+
+You can download the **Service Transfer** form by using the following steps:
+
+**Note** If you are a Private Cloud customer, you can request all of the forms described in this guide from your Account Manager.
+
+**Step 1.** Log in to the `Rackspace Technology Customer Portal <https://login.rackspace.com/>`_.
+
+**Step 2.** In the top navigation panel, select **Account >> Docs and Forms**.
+
+<img width="349" alt="Docs and forms" src="Docs and forms.png">
+
+**Step 3.** In the **Documents and Forms** page, select the appropriate options below **Transfer Ownership to a new Account owner**:
+
+   - Fanatical Support for AWS customer,
+     download the **Fanatical Support for AWS Service Transfer Form**.
+   - Other customers, download the **Service Transfer Form**.
+
+**Step 4.** Complete the downloaded form and save the edited file on your computer.
+
+**Step 5.** Return to the `Rackspace Technology Customer Portal <https://login.rackspace.com/>`_.
+
+**Step 6.** Select **Tickets** > **Create Ticket** from the top navigation panel.
+
+<img width="331" alt="Create Ticket" src="Create Ticket.png">
```

Comment:
update name. to all lower case, no spaces, and use dashes or (underscore if thats what other images have been doing in this repo) instead of spaces.